modules/custom/openy_campaign/src/CampaignMenuService.php

```diff
@@ -358,7 +358,7 @@ public function getActiveCampaigns() {
     $campaignIds = $nodeStorage->getQuery()
       ->condition('type', 'campaign')
       ->condition('status', TRUE)
-      ->condition('field_campaign_end_date', $now->format(DATETIME_DATETIME_STORAGE_FORMAT), '>=')
+      ->condition('field_campaign_end_date', $now->format(\Drupal\datetime\Plugin\Field\FieldType\DateTimeItemInterface::DATETIME_STORAGE_FORMAT), '>=')
```

Comment:
It looks like autogenerated, lets use USE at the top of the file

Reply:
fixed https://github.com/ymcatwincities/openy/pull/2174/commits/839562bfa412cf71c9862ca9d9ce54b72f92ff06